src/otx/config/data/mmdet.yaml

```diff
@@ -0,0 +1,14 @@
+defaults:
+  - default
+
+data_format: coco_instances
```

Comment:
```suggestion
datumaro_data_format: coco_instances
```
Since the filename is the `mmdet.yaml`, users could easily imagine the `data_format` will represents the data format of the `mmdet` instead of `Datumaro`. 

+ If we could add data_format examples, users could easily take the dataset format. (i.e., for my case, I spent some time finding the exact name (imagenet_with_subset_dirs) which has the same functionality `Torchvision.ImageFolder`.